Things.md

```diff
@@ -0,0 +1,139 @@
+```javascript
+// Setting the top bound of a Thing to 10. 
+setTop(thing, 10);
+
+// Setting the left bound of a Thing to 10. 
+setLeft(thing, 10);
+
+// Setting the bottom bound of a Thing to 10. 
+setBottom(thing, 10);
+
+// Setting the right bound of a Thing to 10. 
+setRight(thing, 10);
+```  
+#### Midpoints
+The next is to set the Thing's midpoint using `setMidX`, `setMidY`, and `setMid` (the latter of which sets both the `x` and `y` midpoints). These functions make it so the Thing is centered on the given `x` and `y`.  
+```javascript
+// Setting both x and y midpoints to 1 and 1, respectively. 
+setMid(thing, 1, 1);
+
+// The above is equivalent to the following 2 function calls. 
+setMidX(thing, 1);
+setMidY(thing, 1);
+```
+Going along with setting midpoints, we can do that based on the midpoint of another Thing. Using `setMidXObj`, `setMidYObj`, and `setMidObj` you can shift a Thing so that its horizontal and/or vertical midpoint(s) are centered on that of another Thing.  
+```javascript
+/* Aligning a Thing's horizontal and vertical midpoints with
+ * that of another Thing. This is equivalent to the following
+ * 2 function calls combined.
+ */
+setMidObj(thing, otherThing);
+
+// Aligning just the horizontal midpoint. 
+setMidXObj(thing, otherThing);
+
+// Aligning just the vertical midpoint. 
+setMidYObj(thing, otherThing);
+```
+#### Update
+The next is to use an `update` function (`updateTop`, `updateRight`, `updateBottom`, or `updateLeft`). This is similar to using both `shift` and `set`, where it will shift the Thing's bound by the given offset.
+```javascript
+// Updating the top boundary by shifting it up 1. 
+updateTop(thing, 1);
+
+// Updating the right boundary by shifting if right 1.
+updateRight(thing, 1);
+
+/* Updating the bottom boundary by shifting it up 1.
+ * This is equivalent to the above updateTop function.
+ */
+updateBottom(thing, 1);
+
+/* Updating the left boundary by shifting it right 1.
+ * This is equivalent to the above updateRight function 
+ */
+updateLeft(thing, 1);
+
+```
+#### Slide 
+The next is to use `slideToX` or `slideToY`. These will slide a Thing toward a target `x` or `y`, but also limits the total distance allowed (distance computed from the Thing's original midpoint).  
+```javascript
+// Sliding a Thing to x position 10, moving a max distance of 5. 
+slideToX(thing, 10, 5);
+
+// Sliding a Thing to y position 10, moving a max distance of 5. 
+slideToY(thing, 10, 5);
+```
+
+## Opacity
+Opacity defines how transparent a Thing is. A Thing's opacity is defined in the `opacity` property and ranges from 0 to 1 (1 = fully visible, 0 = fully transparent).  
+You can set a Thing's opacity using `setOpacity`.
+```javascript
+// Setting a Thing's opacity to 1. 
+GameStarter.setOpacity(thing, 1);
+
+// Setting a Thing's opacity to 0 (transparent). 
+GameStarter.setOpacity(thing, 0);
+```
+
+# GroupHoldr
```

Comment:
Looking through the code samples, IMO it would be better to have the GroupHoldr section before the Shifting Things section. You can just cut-and-paste it :)